pkg/istio/rbac/v2/provider_test.go

```diff
@@ -127,7 +127,7 @@ func TestConvertAthenzModelIntoIstioRbac(t *testing.T) {
 			componentsEnabledAuthzPolicy, err := common.ParseComponentsEnabledAuthzPolicy("*")
 			assert.Equal(t, nil, err, "ParseComponentsEnabledAuthzPolicy func should not return nil")
 			p := NewProvider(componentsEnabledAuthzPolicy, true)
-			convertedAuthzPolicy := p.ConvertAthenzModelIntoIstioRbac(domainRBAC, tt.inputService.Name, labels["app"])
+			convertedAuthzPolicy := p.ConvertAthenzModelIntoIstioRbac(domainRBAC, tt.inputService.Name, labels["app"], labels["app"])
```

Comment:
minor: can we update the 3rd arg to be `labels["svc"]` as that matches the real scenario